docs/tutorials/basic-usage.md

```diff
@@ -23,16 +23,16 @@ Basic usage of Shaka Player is very easy:
   <body>
     <video id="video"
            width="640"
-           poster="//shaka-player-demo.appspot.com/assets/poster.jpg"
-           controls autoplay></video>
+           poster="/assets/poster.jpg"
+           controls autoplay playsinline></video>
   </body>
 </html>
 ```
 
 ```js
 // myapp.js
 
-var manifestUri = '//storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd';
+var manifestUri = 'https://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd';
```

Comment:
Can you explain why we shouldn't use // ?  It's not very helpful to make arbitrary changes like this without at least explaining them in the commit message.

Reply:
See https://www.paulirish.com/2010/the-protocol-relative-url/ and https://google.github.io/styleguide/htmlcssguide.html#Protocol. You can also have a look at  b/31773284 and b/73454807

Reply:
Thanks!  Can you please update the other instances of protocol-relative URIs as well?  There are many, many of these in the code.